src/applications/personalization/profile-2/components/MobileMenuTrigger.jsx

```diff
@@ -53,6 +53,7 @@ const MobileMenuTrigger = ({
 
       if (justSwitchedToMobile) {
         setIsMobile(true);
+        setTriggerHeight(window.getComputedStyle(button.current).height);
```

Comment:
The height of the trigger was getting set to `0` component mount (line 36) if the screen wasn't narrow enough. So we need to set it here, when the screen switches from mobile width for the first time.